chainer/functions/array/hstack.py

```diff
@@ -53,11 +53,31 @@ def hstack(xs):
     """Concatenate variables horizontally (column wise).
 
     Args:
-        xs (list of chainer.Variable): Variables to be concatenated.
+        xs (list of :class:`~chainer.Variable` or :class:`numpy.ndarray` or \
+        :class:`cupy.ndarray`):
+            Input variables to be concatenated. The variables must have the
+            same shape, except in the horizontal (column wise) dimension.
```

Comment:
I recognized my misunderstand. I will check the result and write the descriptions.